<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageAbout</class>
 <widget class="QWidget" name="PageAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>582</height>
   </rect>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout" stretch="0">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="TabLayout" name="tblAbout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInfo">
       <attribute name="title">
        <string>Informations</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>411</width>
          <height>481</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layTabInfo">
         <item>
          <widget class="QTextBrowser" name="txtInfo">
           <property name="source">
            <url>
             <string>qrc:/about.html</string>
            </url>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabTnx">
       <attribute name="title">
        <string>Thanks</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>411</width>
          <height>481</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layTabTnx">
         <item>
          <widget class="QTextBrowser" name="textBrowser">
           <property name="source">
            <url>
             <string>qrc:/thanks.html</string>
            </url>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLicense">
       <attribute name="title">
        <string>License</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>411</width>
          <height>481</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layTabLicense">
         <item>
          <widget class="QTextBrowser" name="txtLicense">
           <property name="source">
            <url>
             <string>qrc:/license.html</string>
            </url>
           </property>
           <property name="searchPaths">
            <stringlist notr="true"/>
           </property>
           <property name="openLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabLayout</class>
   <extends>QTabWidget</extends>
   <header>tablayout.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
